### PR TITLE
Practice PR

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

This is a practice PR

### How to Review

[//]: # 'The order in which to review files and what to expect when testing locally'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

[//]: # "What doesn't work yet, what's NOT in this PR/has to be done "

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # People Power Web App
 
-_Last updated: Feb 19, 2019 (by Aivant)_
+_Last updated: Feb 19, 2019 (by Cindy)_
```
Comment (aivantg) [inline README.md]: Hey why did you do this?
